services/validators/envValidator.go

```diff
@@ -124,7 +124,7 @@ func getWorkspaceFromDB(dbHandle *sql.DB) (string, error) {
 }
 
 func createDBConnection() (*sql.DB, error) {
-	psqlInfo := misc.GetConnectionString(config.Default)
+	psqlInfo := misc.GetConnectionString(config.Default, "env-validator")
```

Comment:
`envValidator` is killing dangling connections during startup based on the `APPLICATION_NAME`. Now that we are not using the exact same `APPLICATION_NAME` across rudder-server, we need to adapt the query used in `killDanglingDBConnections` accordingly.

Reply:
updated query in killDanglingDBConnections 👍